AsyncDisplayKit/ASViewController.mm

```diff
@@ -82,6 +83,13 @@ - (void)loadView
   _node.frame = frame;
   _node.autoresizingMask = autoresizingMask;
   self.view = view;
+  
+  // ensure that self.node has a valid trait collection before a subclass's implementation of viewDidLoad.
+  // Any subnodes added in viewDidLoad will then inherit the proper environment.
```

Comment:
This seems worthwhile as a performance optimization. However, both for clarity of my understanding and of the description, would it not be the case that Setting a trait collection after the children are added would trigger a propagation of the new values?